lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -108,7 +104,7 @@ def instructor_dashboard_2(request, course_id):
         'studio_url': get_studio_url(course, 'course'),
         'sections': sections,
         'disable_buttons': disable_buttons,
-        'analytics_dashboard_message': analytics_dashboard_message
+        'analytics_dashboard_url': analytics_dashboard_url,
```

Comment:
analytics_dashboard, i.e. Insights, is a bit confusing as there's another tab named Analytics. Any way you can provide more clarity?